External/AvalonTools/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
 if(RDK_BUILD_AVALON_SUPPORT)
+set(AVALONTOOLS_DIR "$ENV{AVALONTOOLS_DIR}") 
```

Comment:
It must have been a leftover of some early testing that I did - that's
absolutely unnecessary, I hadn't realized it was there. I have removed that
line. Thanks for pointing out.

On Thu, Feb 4, 2016 at 5:42 AM, Greg Landrum <notifications@github.com>
wrote:

> In External/AvalonTools/CMakeLists.txt
> <https://github.com/rdkit/rdkit/pull/752#discussion_r51831817>:
>
> > @@ -1,4 +1,8 @@
> >  if(RDK_BUILD_AVALON_SUPPORT)
> > +set(AVALONTOOLS_DIR "$ENV{AVALONTOOLS_DIR}")
>
> This block of changes screws up subsequent runs of cmake in an already
> existing build directory where RDK_BUILD_AVALON_SUPPORT is already set if
> there is no AVALONTOOLS_DIR variable in the active environment.
>
> Why do you think these are necessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rdkit/rdkit/pull/752/files#r51831817>.
>